Formula/varnish.rb

```diff
@@ -5,7 +5,7 @@ class Varnish < Formula
   mirror "https://fossies.org/linux/www/varnish-7.0.0.tgz"
   sha256 "8c7a5c0b1f36bc70bcbc9a48830835249e895fb8951f0363110952148cbae087"
```

Comment:
https://varnish-cache.org/releases/
## Line 4 would also need to be updated to v7.0.1
```suggestion
  mirror "https://fossies.org/linux/www/varnish-7.0.1.tgz"
  sha256 "c4e75beff0d461611742361fe8039ee1233ddf755b2b8a1e18a5fcacbe2b4660"
```